.github/workflows/commit-and-tag.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+set -e
+
+PACKAGE_VERSION=$1
+GPG_KEY_ID=$2
+
+gpgloader
+
+# Create signed "Package x.y.z" commit
+echo "Create package commit"
+git commit -m "Package ${PACKAGE_VERSION}" -s --gpg-sign=${GPG_KEY_ID} phongo_version.h
+
+# Create signed "Release x.y.z" tag
+echo "Create release tag"
+git tag -F changelog -s --local-user=${GPG_KEY_ID} ${PACKAGE_VERSION}
```

Comment:
The `changelog` file is hard-coded here, which couples this script to `release.yml`. Should it be a parameter instead? Even having it defined as a script var (e.g. `TAG_MESSAGE=changelog`) would improve readability.

Reply:
Ah, yes. I introduced a `TAG_MESSAGE_FILE` argument to `commit-and-tag.sh`